contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -4627,4 +4639,38 @@ static Node* optimize_explicit_cast(ParseState *pstate, Node *node)
 		}
 	}
 	return node;
+}
+
+static void babelfixedparallelstate_insert(ParallelContext *pcxt, bool estimate)
+{
+	BabelfishFixedParallelState *bfps;
+	int len;
+	char* current_db_name;
+	if (estimate)
+	{
+		/* Allow space to store the babelfish fixed-size parallel state. */
+		shm_toc_estimate_chunk(&pcxt->estimator, sizeof(BabelfishFixedParallelState));
+		shm_toc_estimate_keys(&pcxt->estimator, 1);
```

Comment:
what is meaning of 1 here?

Reply:
The second argument signified the number of keys we want to insert. In our case it is one, BABELFISH_PARALLEL_KEY_FIXED, hence 1.